src/main/java/core/basesyntax/dao/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.dao.impl;
+
+import core.basesyntax.dao.FileWriter;
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String filePath, String data) {
+        try (BufferedWriter writer = new BufferedWriter(new java.io.FileWriter(filePath))) {
```

Comment:
According to the checklist, we should avoid hardcoding file paths. It's better to pass the file path as a parameter to the method which is done correctly here. However, ensure that the file path is never hardcoded elsewhere in the application.